fhir-server-test/src/test/java/testng.xml

```diff
@@ -1,43 +1,53 @@
-            <class name="com.ibm.fhir.server.test.UpdateTest" />
-            <class name="com.ibm.fhir.server.test.DuplicateResourceIdTest"/>
-            <class name="com.ibm.fhir.server.test.FHIRDocumentOperationTest" />
-            <class name="com.ibm.fhir.server.test.FHIRValidateOperationTest" />
-            <class name="com.ibm.fhir.server.test.FHIRHealthcheckOperationTest" />
-            <class name="com.ibm.fhir.server.test.PrettyServerFormatTest" />
-            <class name="com.ibm.fhir.server.test.MultiDataStoreTest" />
-            <class name="com.ibm.fhir.server.test.SearchTest" />
-            <class name="com.ibm.fhir.server.test.SearchAllTest" />
-            <class name="com.ibm.fhir.server.test.SearchExtensionsTest" />
-            <class name="com.ibm.fhir.server.test.SortingTest" />
-            <class name="com.ibm.fhir.server.test.SearchNearTest" />
-            
-            <class name="com.ibm.fhir.client.test.FHIRClientTest" />
-
-            <class name="com.ibm.fhir.cli.test.FHIRCliTest" />
-            <!-- requires $hello operation to be deployed. 
-            Include the fhir-operation-[version]-tests jar in the fhir-server userlib dir before running the test:
-            -->
-            <class name="com.ibm.fhir.server.test.FHIROperationTest" />
-            
-            <!-- The following class is intentionally moved to last.  
-                During debug, the other tests interfere with the asynch nature of the test -->
-            <class name="com.ibm.fhir.server.test.WebSocketNotificationsTest" />
-        </classes>
-    </test>
-</suite>
+        <packages>
+            <!-- All of these tests are automatically executed -->
+            <package name="com.ibm.fhir.server.test" />
+        </packages>
+    </test>
+    <test name="ClientTests">
+        <packages>
+            <!-- All of these tests are automatically executed -->
+            <package name="com.ibm.fhir.cli.test" />
+            <package name="com.ibm.fhir.client.test" />
+            <package name="com.ibm.fhir.client.test.oauth" />
+        </packages>
+    </test>
+    <test name="BulkDataTests">
+        <packages>
+            <!-- All of these Bulk Data tests are automatically executed -->
+            <package name="com.ibm.fhir.server.test.bulkdata" />
+        </packages>
+    </test>
+    <test name="Db2Tests">
+        <packages>
+            <!-- All of these Db2 tests are automatically executed -->
+            <package name="com.ibm.fhir.server.test.db2" />
+        </packages>
+    </test>
+    <test name="KafkaTests">
+        <packages>
+            <!-- All of these Kafka tests are automatically executed -->
+            <package name="com.ibm.fhir.server.test.kafka" />
+        </packages>
+    </test>
+    <test name="WebsocketTests">
+        <packages>
+            <!-- All of these websocket tests are automatically executed -->
+            <package name="com.ibm.fhir.server.test.websocket" />
```

Comment:
Given that `test.websocket.enabled = false` by default, I think this comment is misleading.
Technically it is "executed" but we skip all the logic, right?

Reply:
It is executed locally, on the server it is run, let me clarify this in the comment

Reply:
Clarified this actually, it's disabled = false by default.